src/lib/components/Carousel/imageCarousel.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+	import type { ImageMetadata } from '$lib/types/ImageCarousel';
+
+	export let images: ImageMetadata[] = [];
+  export let carouselId: string;
+</script>
+
+<div id={carouselId} class="carousel slide carousel-fade relative pb-4" data-bs-ride="carousel">
+	<!-- create slideshow at the bottom of the carousel users can click through -->
+	<div
+		class="carousel-indicators absolute right-0 bottom-0 left-0 flex justify-center p-0 mb-4 pb-4"
+	>
+		{#each images as _, i}
+			{#if i == 0}
+				<button
+					type="button"
+					data-bs-target={`#${carouselId}`}
+					class="active"
+					aria-current="true"
+					data-bs-slide-to={0}
```

Comment:
Unfortunately the 1st button has slightly different logic to the other buttons. The [library](https://tailwind-elements.com/docs/standard/components/carousel/) needs to know which button is active first - lines 19 & 20 are unique to the 1st button